src/components/Login/Login.jsx

```diff
@@ -0,0 +1,64 @@
+import axios from 'axios';
+import React from 'react';
+import { Link, useNavigate } from 'react-router-dom';
+import styles from '../Login/Login.module.css';
+
+import Button from '../../common/Button/Button';
+import Input from '../../common/Input/Input';
+
+const Login = () => {
+  const navigate = useNavigate();
+
+  const login = (event) => {
+    event.preventDefault();
+
+    axios
+      .post('http://localhost:4000/login', {
+        name: event.target.elements.name.value,
+        email: event.target.elements.email.value,
+        password: event.target.elements.password.value,
+      })
+      .then((response) => {
+        localStorage['username'] = response.data.user.name;
+        localStorage['userToken'] = response.data.result;
+        navigate('/courses');
+      })
+      .catch((error) => {
+        console.log(error);
+        console.log(error.response);
+        console.log(error.message);
+        console.log(error.request);
+      });
```

Comment:
Logging of `error` would be enough :)
Also, please, use `console.error` for better logging flow